spec/index.html

```diff
@@ -816,7 +833,7 @@ <h4 id="commit"><a class="toc-anchor" href="#commit" >§</a> Commit</h4>
 <li>The message object <strong><strong>MUST</strong></strong> contain a <code>descriptor</code> property, and its value <strong><strong>MUST</strong></strong> be a JSON object composed as follows:
 <ul>
 <li>The object <strong><strong>MUST</strong></strong> contain a <code>method</code> property, and its value <strong><strong>MUST</strong></strong> be the string <code>CollectionsCommit</code>.</li>
-<li>The object <strong><strong>MUST</strong></strong> contain an <code>recordId</code> property, and its value <strong><strong>MUST</strong></strong> be a [<a class="spec-reference" href="#ref:RFC4122">RFC4122</a>] UUID Version 4 string.</li>
+<li>The object <strong><strong>MUST</strong></strong> contain a <code>recordId</code> property, and its value <strong><strong>MUST</strong></strong> be a [<a class="spec-reference" href="#ref:RFC4122">RFC4122</a>] UUID Version 4 string.</li>
```

Comment:
As above, `a RFC4122` vs `an RFC4122`... I think the latter is correct.
```suggestion
<li>The object <strong><strong>MUST</strong></strong> contain a <code>recordId</code> property, and its value <strong><strong>MUST</strong></strong> be an [<a class="spec-reference" href="#ref:RFC4122">RFC4122</a>] UUID Version 4 string.</li>
```